modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -11001,10 +11001,12 @@ object=Object
 object-action-executor-help[add-object-entry]=Add entry on related or unrelated objects.
 object-action-executor-help[groovy]=Create custom actions using Groovy.
 object-action-executor-help[notification]=Send a notification template.
+object-action-executor-help[update-object-entry]=Update the current entry
```

Comment:
@marcelabc This pull request LGTM, but maybe Brian will ask If this a full sentence (the end of a complete sentence should be marked by a period..) or a Title (capitalize the beginning of every word (?)) 🤦🏽‍♀️